test/local/isolated-pool-multi-users-utilisation-rates.test.ts

```diff
@@ -53,7 +53,7 @@ describe('Pool with fixed interests rates', () => {
 
       await pool.connect(user1).withdraw(toWei("5.68"));
 
-      depositUser1 = fromWei(await pool.getDeposits(user1.address));
+      depositUser1 = fromWei(await pool.balanceOf(user1.address));
```

Comment:
**In line 63:**
What does this check actually check? `borrowedUser2` seems to be the 'old' value (before `repay`) that is greater than 6 and at the same time `depositUser1` was refreshed after the `repay` operation

Reply:
sorry, this test shouldn't be included in commit. Removing. This logic is tested in `pool-health.test.ts`